base/src/middleware/http_errors.mustache

```diff
@@ -0,0 +1,18 @@
+const HttpError = require('standard-http-error');
+const {
+  thinky: {
+    Errors: { DocumentNotFound, ValidationError }
+  }
+} = require('../../config');
+
+module.exports = (err, req, res, next) => {
+  if (err instanceof DocumentNotFound) {
+    res.status(404).send({ error: 'not found' });
```

Comment:
i think creating a binding like that between tests in code in integration tests is dodgy and defies the point of integration tests. if someone screws up the config the tests will keep passing